build.gradle

```diff
@@ -10,20 +10,19 @@ buildscript {
    dependencies {
       classpath "us.ihmc:ihmc-pub-sub-generator:0.14.0"
       classpath 'com.halodi:halodi-csharp-pub-sub-generator:0.1.1'
-      classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.+'
    }
    
 
 }
 
 apply plugin: 'java'
-apply plugin: 'com.jfrog.bintray'
 apply plugin: 'maven'
 apply plugin: 'maven-publish'
 
 
 group = "com.halodi"
 version = "1.5.12"
```

Comment:
even version can be an env variable that is pulled from the git tag

Reply:
I totally agree, but that was not what I was thinking of changing in this PR.

Reply:
i'm getting ahead of myself 🤦